src/inmanta/data/__init__.py

```diff
@@ -4157,6 +4157,7 @@ async def query_resource_actions(
         first_timestamp: Optional[datetime.datetime] = None,
         last_timestamp: Optional[datetime.datetime] = None,
         action: Optional[const.ResourceAction] = None,
+        resource_id: Optional[ResourceIdStr] = None,
```

Comment:
I don't like this method breaking up the id's so I updated it to be able to handle complete id's